src/main/kotlin/com/manimdsl/errorhandling/ErrorHandler.kt

```diff
@@ -18,6 +18,15 @@ object ErrorHandler {
         semanticErrors.add("Semantic error at $linePos: $errorEvent")
     }
 
+    fun addRuntimeError(errorEvent: String, lineNumber: Int) {
+        println(
+                "Error detected during program execution. Animation could not be generated. \n" +
```

Comment:
If we have a "\n" then we might as well just println again instead of explicitly saying "\n" and +